src/auto-text-size-react.tsx

```diff
@@ -19,17 +20,22 @@ export function AutoTextSize({
   fontSizePrecisionPx,
   as: Comp = "div", // TODO: The `...rest` props are not typed to reflect another `as`.
   children,
+  onUpdate = () => {},
   ...rest
 }: Options & {
   as?: keyof ReactHTML | React.ComponentType<any>;
+  onUpdate?(): void;
 } & DetailedHTMLProps<
     HTMLAttributes<HTMLDivElement>,
     HTMLDivElement
   >): ReactElement {
   const updateTextSizeRef = useRef<ReturnType<typeof autoTextSize>>();
+  const onUpdateRef = useRef<typeof onUpdate>(onUpdate);
 
   useEffect(() => updateTextSizeRef.current?.(), [children]);
 
+  useImperativeHandle(onUpdateRef, () => onUpdate)
```

Comment:
i'm fine with adding `useCallback`, i just used a ref to keep the callback updated because a lot of people don't have the habit to use it, but maybe we can just add a warning in the README and pass this responsibility to the user

Reply:
yeah, both the `useEffect` and `useImperativeHandle` approaches do the same in this case, but I can change to `useEffect` if you want, no problem

Reply:
hey @ViktorQvarfordt, sorry for the huge delay, I've just updated the branch with the origin one, let me know if you still want me to change the `useImperativeHandle` call.

Reply:
Yeah, most people aren't familiar with this hook. Simple is good!

Reply:
done!